packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -267,33 +263,44 @@ describeNoCompat("GC Sweep tests", (getTestObjectProvider) => {
              * At this point the test has finished, we might want to store a list of all the actions at this point and await them all to make sure they finish.
              * Feel free to put anything in the debug object. It's likely to get big. You can also put a range of actions if you're just trying to hone in on
              * what changed.
+             *
+             * This is the validation and data storage part of the test.
              */
-            const debugObject = {
+
+            const stats = {
+                ...actionStats,
                 actionCount: actionsList.length,
-                actionsList,
                 errorCount: errorList.length,
-                errorList,
-                ...actionStats,
-                handleRecord: handleTracker.handleActionRecord,
             };
 
+            fs.mkdirSync(`nyc/testData-${seed}`, { recursive: true });
+            fs.writeFileSync(`nyc/testData-${seed}/events.json`, JSON.stringify(overrideLogger.events));
+            fs.writeFileSync(`nyc/testData-${seed}/inactiveObjectEvents.json`, JSON.stringify(overrideLogger.inactiveObjectEvents));
+            fs.writeFileSync(`nyc/testData-${seed}/errorEvents.json`, JSON.stringify(overrideLogger.errorEvents));
+            fs.writeFileSync(`nyc/testData-${seed}/errorEventStats.json`, JSON.stringify(overrideLogger.errorEventStats));
+            fs.writeFileSync(`nyc/testData-${seed}/actions.json`, JSON.stringify(actionsList));
+            fs.writeFileSync(`nyc/testData-${seed}/stats.json`, JSON.stringify(stats));
+            fs.writeFileSync(`nyc/testData-${seed}/errors.json`, JSON.stringify(errorList));
```

Comment:
Might be nice to wrap some of these in an uber-object and log that to a single file?  Up to you, I haven't actually worked with the process/data.